src/Configuration.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * Common configurations for Proxy PagoPa and external resources
+ */
+
+import * as t from "io-ts";
+import { WithinRangeNumber } from "italia-ts-commons/lib/numbers";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+
+const localhost = "http://localhost";
+
+export const CONFIG = {
+  /* RESTful Webservice configuration
```

Comment:
use `//` syntax for inline comments or 

```
/** 
  *
  */
```
for block comments

(it makes easier to grep out comments)